lib/matplotlib/axes/_axes.py

```diff
@@ -4247,13 +4247,16 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
             is 'face'. You may want to change this as well.
             If *None*, defaults to rcParams ``lines.linewidth``.
 
-        edgecolors : color or sequence of color, optional, default: 'face'
+        edgecolors : {'face', 'none', *None*} or color, optional
```

Comment:
please also leave the "sequence of colors" in. You can do:

~~~
plt.scatter([0, 1, 2], [1, 3, 2], s=400, marker='o', edgecolors=['c', 'm', 'y'])
~~~